SCANNER_VERSION

```diff
@@ -1 +1 @@
-2.24.0
+2.24.0-11-g05cf175999
```

Comment:
Version is from the build based on this commit: https://github.com/stackrox/scanner/commit/05cf1759997f2dd0a831b1745aefed94d5ab16c3